### ISSUE TITLE
V and DVM

### ISSUE THREAD

**Issue by coreyb42** [derek73/python-nameparser#36]
Both HumanName('Hardie Leannon V') and HumanName('Hardie Leannon DVM') fail, improperly assigning 'V' and 'DVM' to the last name, respectively.

```
>>> HumanName('Hardie Leannon V')
<HumanName : [
	title: '' 
	first: 'Hardie' 
	middle: 'Leannon' 
	last: 'V' 
	suffix: ''
	nickname: ''
]>
>>> HumanName('Hardie Leannon DVM')
<HumanName : [
	title: '' 
	first: 'Hardie' 
	middle: 'Leannon' 
	last: 'DVM' 
	suffix: ''
	nickname: ''
]>
```

**Comment by derek73** [comment]
Could you also add the output you expect for each example?

**Comment by derek73** [comment]
Do you think both the end pieces should be suffixes? DVM we could solve if that's a title/suffix, but V could also be an initial.

**Comment by coreyb42** [comment]
DVM is definitely a title, much like MD (DVM stands for Doctor of Veterinary Medicine), so I would expect that it would go into title. Good point on the V - it is a Roman numeral at the end, so I would expect it to go into suffix. It is plausible that it could be an initial, but I would say it's more likely to be a numeral since the other two words are spelled out.

**Comment by derek73** [comment]
I think I fixed both of these in v0.3.8. I had been meaning to try using a regex for roman numerals at the end of a name.
